_posts/2015-06-03-team-offsite.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Team ePages 6 offsite event"
```

Comment:
I agree with Kristof, for external use, team ePages 6 means nothing. I would go with a functional name like "ePages Core Team" or something along those lines. Even a colour is quite meaningless